sdk/identity/identity/src/credentials/managedIdentityCredential/models.ts

```diff
@@ -14,6 +14,19 @@ export interface MSIConfiguration {
   resourceId?: string;
 }
 
+/**
+ * @internal
+ * Represents an access token for {@link ManagedIdentity} for internal usage,
+ * with an expiration time and the time in which token should refresh.
+ */
+export declare interface MSIToken extends AccessToken {
+  /**
+   * The time in which token should refresh,
+   * specified in milliseconds, UNIX epoch time.
+   */
+  refreshesIn?: number;
```

Comment:
Is this a millisecond _offset_ from when the token was issued, or an epoch timestamp? The name implies it is some kind of duration ("refreshes in 36000000 milliseconds") but the doc string sounds like it's saying this is a time _stamp_. The code suggests to me that it's a time stamp and so maybe it should be called "refreshesOn."

Reply:
so the service returns to us as a "second" offset from when the token was issued and we for out internal purpose convert it to a timestamp from epoch (in milliseconds) and then when we need to send it to msal back, we convert it back to seconds